src/cli/commands/test/index.ts

```diff
@@ -105,6 +105,19 @@ export default async function test(
   let iacScanFailures: IacFileInDirectory[] | undefined;
   let iacOutputMeta: IacOutputMeta | undefined;
 
+  const isNewIacOutputSupported = options.iac
+    ? await hasFeatureFlag('iacCliOutput', options)
+    : false;
+
+  if (shouldPrintIacInitialMessage(options, isNewIacOutputSupported)) {
+    console.log(
+      EOL +
+        chalk.bold(
```

Comment:
I've examined the coloration of the message and got the result in the image attached below.
I think we need to add the `white` context for `chalk` here.

```suggestion
        chalk.white.bold(
```

In the image below, the top message is using the `white` context and the bottom one does not:

![image](https://user-images.githubusercontent.com/46415136/161591943-964c5f4b-f8ec-4697-8568-d9f9088a1575.png)

Reply:
Fixed at f3475a97 and updated the screenshot in the description.